cluster/gce/util.sh

```diff
@@ -3691,12 +3693,12 @@ function check-resources() {
   echo "Looking for already existing resources"
   KUBE_RESOURCE_FOUND=""
 
-  if [[ -n "${INSTANCE_GROUPS[@]:-}" ]]; then
-    KUBE_RESOURCE_FOUND="Managed instance groups ${INSTANCE_GROUPS[@]}"
+  if [[ -n "${INSTANCE_GROUPS[*]:-}" ]]; then
```

Comment:
why?

Reply:
> why?

https://github.com/koalaman/shellcheck/wiki/SC2199

I believe what we want to achieve here is the

`Alternatively, if you do want to concatenate all the values in the array into a single string for your test, use "$*" or "${array[*]}" to make this explicit.
`

usecase they describe.